dev/provision_salt.sh

```diff
@@ -22,14 +22,27 @@ function install_salt() {
   elif [[ "$OS" == 'Linux' ]]; then
     DISTRO="$(lsb_release -is)"
     if [[ "$DISTRO" == 'Ubuntu' ]]; then
-      # Download the SALT client tarball
-      SALT_DEB_PKG="/tmp/salt.deb"
-      curl -L -o "$SALT_DEB_PKG" https://packages.broadcom.com/artifactory/saltproject-deb/pool/salt-api_3006.9_amd64.deb
-      # Uninstall previous package, if any
-      sudo dpkg --remove salt-minion
-
-      # Install our downloaded .deb package
-      sudo dpkg --install "$SALT_DEB_PKG"
+      # Uninstall previous SaltStack packages, if any.
+      sudo dpkg --remove salt-minion salt-api salt-common
```

Comment:
Is this resilient to if the packages aren't already installed?

Reply:
Tested locally, yep, it appears to exit 0.

Reply:
Yes indeed